shell-completion/bash/systemd-cgtop

```diff
@@ -56,6 +56,9 @@ _systemd_cgtop() {
     fi
 
     COMPREPLY=( $(compgen -W '${OPTS[*]}' -- "$cur") )
+    if [ -d /sys/fs/cgroup/ ]; then
+        COMPREPLY+=( $(cd /sys/fs/cgroup/ && compgen -o nospace -o dirnames "$cur") )
```

Comment:
This does not work with cgroup v1. Do we care?

Reply:
Speaking for myself, I really don't :-) I can add a quick check to ensure it's in unified mode?

Reply:
Maybe try to cd to /sys/fs/cgroup/systemd first, and then /sys/fs/cgroup otherwise?

Reply:
ok, done